eaf.el

```diff
@@ -687,6 +744,7 @@ Use it as (eaf-bind-key var key eaf-app-keybinding)"
             (when (and
                    (derived-mode-p 'eaf-mode)
                    (equal eaf--buffer-id bid))
+              (setq-local eaf--browser-full-title title)
```

Comment:
Not just browser title, we should use eaf--full-title

Reply:
Okay